examples/chip-tool/commands/echo/Commands.h

```diff
@@ -35,8 +35,14 @@ class EchoBle : public EchoCommand
 
 void registerCommandsEcho(Commands & commands)
 {
-    commands.Register(make_unique<Echo>());
-    commands.Register(make_unique<EchoBle>());
+    const char * clusterName = "";
```

Comment:
should this be echo?

Reply:
Not really. This is an empty name until the echo command is really part of a Cluster. See https://github.com/project-chip/connectedhomeip/pull/3020